### ISSUE TITLE
Size Scales

### ISSUE THREAD

**Issue by zehfernandes** [vtex-apps/vtex-tachyons#4]
A escala `1` e `2` possuem uma variação pequena o que nos dá batante controle.
Mas ao pular para a `3` o salto é muito maior.

Sugiro em todos as unidades de medida preenchermos com escalas intermediárias nos breakpoints acima de 2.

indo até a medida `10`.

**Comment by dfosco** [comment]
Concordo! Como fazer essa customização? Na mão ou você quer automatizar alguma coisa no CLI?
